ui/src/components/Terminal/index.vue

```diff
@@ -84,6 +84,8 @@ const receivePostMessage = (): void => {
         emits('update:drawer', true, t('Settings'), 'setting');
         break;
       case WINDOW_MESSAGE_TYPE.FILE:
+        emits('update:drawer', true, t('FileManager'), 'file-manager');
+        break;
       case WINDOW_MESSAGE_TYPE.FOCUS:
         terminal.value?.focus();
         break;
```

Comment:
The provided code snippet does not have significant irregularities, potential issues, or optimization suggestions. It appears to be written correctly based on the specifications of what needs to be done with the window messages.

If you need help in another area, please specify so I can assist further.